record-builder-processor/src/main/java/io/soabase/recordbuilder/processor/ElementUtils.java

```diff
@@ -86,8 +86,9 @@ public static String getPackageName(TypeElement typeElement) {
                 break;
             }
         }
-        String name = typeElement.getEnclosingElement().toString();
-        return !name.equals("unnamed package") ? name : "";
+        String name = typeElement.getQualifiedName().toString();
```

Comment:
We lost the check for "unnamed package" - is that no longer needed?

Reply:
Indeed, it's no longer required: the qualified name of a *typeElement* is the only name of the element when inside the default/unnamed package while it's the fully qualified name (dot separated) of the package + name of the *typeElement* otherwise (e.g. `a.b.c.MyClass`). 

The dot separated qualified name is part of the contract of the `javax.lang.model.element.Element` class, contrary to the output of the `toString()` method.